### PR TITLE
Adding a display field of parent requisition

### PR DESCRIPTION
**Pull request by yvaucher**
This allows to use the same view for both line form view
and inner form view in requisition of line

### CODE REVIEW

File: logistic_requisition/model/logistic_requisition.py [hunk 1]
```diff
@@ -333,6 +333,13 @@ class LogisticRequisitionLine(models.Model):
         readonly=True,
         required=True,
         ondelete='cascade')
+    # field for displaying requisition both on
+    # line form view and on requisition on inner line form view
+    display_requisition_id = fields.Many2one(
+        comodel_name='logistic.requisition',
+        string='Requisition',
+        compute='_get_requisition_id',
```
Comment (gurneyalex) [inline logistic_requisition/model/logistic_requisition.py]: i'd advocate for a stricter naming convention here: "_get_display_requision_id"

File: logistic_requisition/model/logistic_requisition.py [hunk 2]
```diff
@@ -333,6 +333,13 @@ class LogisticRequisitionLine(models.Model):
...
+        readonly=True)
```
Comment (gurneyalex) [inline logistic_requisition/model/logistic_requisition.py:342]: is there a performance hit when using this in the view?

Comment (yvaucher) [inline logistic_requisition/model/logistic_requisition.py:342]: You only load this in form view it doesn't appear in tree view. So it shouldn't be an issue with many records. In form view it should have almost no impact.

**Comment by yvaucher** [comment]
Further note to explain this case. I cannot use active_id to set requisition_id as it would say me:

```
The following fields are invalid:
Requisition
```

File: logistic_requisition/model/logistic_requisition.py (hunk 2 above)
Comment (gurneyalex) [inline logistic_requisition/model/logistic_requisition.py:342]: ok then. Thanks for the feedback

**Comment by yvaucher** [comment]
@gurneyalex fixed getter method name

**Comment by gurneyalex** [comment]
:+1:

**Comment by lepistone** [comment]
thanks :+1:

**Comment by foutoucour** [comment]
:+1:

**Pull request merged**